dev/create-release/spark-rm/Dockerfile

```diff
@@ -38,7 +38,8 @@ ARG PIP_PKGS="pyopenssl pypandoc numpy pygments sphinx"
 #
 # This is all in a single "RUN" command so that if anything changes, "apt update" is run to fetch
 # the most current package versions (instead of potentially using old versions cached by docker).
-RUN echo 'deb http://cran.cnr.Berkeley.edu/bin/linux/ubuntu xenial/' >> /etc/apt/sources.list && \
+RUN apt-get clean && apt-get update && $APT_INSTALL gnupg ca-certificates apt-transport-https && \
```

Comment:
This also needed `apt-transport-https` in Ubuntu 16, it seems. I tried the steps in Docker and it successfully accesses CRAN.